MODULE.bazel

```diff
@@ -0,0 +1,7 @@
+module(
+    name = "google_toolbox_for_mac",
+    version = "3.0.0",
```

Comment:
I assume we'll need to update this when we do releases and bump the version info in cocoapods?

Reply:
Yes, indeed; that's my understanding.